controllers/utils_test.go

```diff
@@ -164,6 +164,14 @@ func getTestPolicyServerService(policyServerName string) (*corev1.Service, error
 	return &service, nil
 }
 
+func getTestPolicyServerCASecret() (*corev1.Secret, error) {
```

Comment:
This is retrieving the Root CA created by our controller. I suggest to rename this helper function to reflect that.

Maybe something like `getInternalRootCASecret` ?